orm/src/test/java/org/hibernate/search/test/query/dsl/DSLTest.java

```diff
@@ -890,6 +890,57 @@ public void testUseMatchQueryOnEmbeddedNumericIdCreatesTermQuery() throws Except
 		);
 	}
 
+	@Test
+	@TestForIssue(jiraKey = "HSEARCH-1793")
+	public void testQueryingForEntitiesFromDifferentIdSpaces() throws Exception {
+		QueryBuilder queryBuilder = fullTextSession.getSearchFactory()
+				.buildQueryBuilder().forEntity( EducationalInstitution.class ).get();
+
+		Query query = queryBuilder.bool()
```

Comment:
I think some comments either here at the query construction or at the assert step would help. Why does this setup and the result you are asserting verify that HSEARCH-1793?